src/System.Private.CoreLib/src/System/Threading/WaitHandle.cs

```diff
@@ -448,7 +417,7 @@ public static int WaitAny(WaitHandle[] waitHandles, int millisecondsTimeout)
 
             //NOTE: This API is not supporting Pause/Resume as it's not exposed in CoreCLR (not in WP or SL)
             int ret = SignalAndWaitOne(toSignal._waitHandle, toWaitOn._waitHandle, millisecondsTimeout,
```

Comment:
Delete the thread affinity argument completely?

Reply:
My next step is basically replacing the whole `WaitHandle.cs` with the version from CoreRT moved to shared (+/- few changes). It didn't seem worth any further cleanup, the next PR will do that.

Reply:
Ah, on second thought I left it in the InternalCall APIs, I will address that.